### DISCUSSION TITLE
yarn bin and lib doesn't match

### DISCUSSION THREAD

**Discussion by leonprou** [yarnpkg/berry#5456] [Q&A]
I'm installing yarn with `npm i -g --verbose yarn@1.22.19` and while in `lib`  getting the right version, the `bin` directory gets version 3:

```
yarn --version
3.1.0
```

I tracked the issue to the tar library, so I was able to recreate the following:

```
wget https://registry.npmjs.org/yarn/-/yarn-1.22.19.tgz
./bin/yarn -v //  OUTPUT:  3.1.0
cat package.json | grep version // OUTPUT: "version": "1.22.19"
```

**Comment by iNViTiON** [comment]
Where did you execute `yarn -v` command? Is it in another project that set local project Yarn version to 3.1.0?

**Reply by leonprou** [reply]
Hi @iNViTiON,
It was globally, not in a project dir.

I was able to fix the issue! I forgot to run `corepack enable`.
Now it works, thanks!
